### ISSUE TITLE
Touch not working on touch devices

### ISSUE THREAD

**Issue by AlvSovereign** [FrDH/jQuery.TosRUs#71]
Hi all

I'm having issues with enabling the touch functionality on the website I'm coding. Please go here: http://lhip.co.uk/gallery.html.

Please click on any of the anchor tags to bring up the light box.

A few things to note: I have included the 'all.css' as an import to the 'main.css' file, which is linked in the head of the page.

I am linking to TosRus.all.min.js, FlameViewportScale.js, and both hammer.min.js and jquery.hammer.js. When I insert:

$(function() {
  alert( $.tosrus.support.touch );
});

to the concole on  Chrome, I am alerted 'true'.

FrDH suggested a solution where we insert:

if ( typeof Hammer != 'function' ) {
    return;
}

somewhere. Where do I need to enter this?

Any ideas please?

Thank you for your time.

AlvSovereign

**Comment by FrDH** [comment]
What version of Hammer are you using? The later versions of tosrus require Hammer version 2 or newer. Also, I think it doesn't need the jquery wrapper for Hammer.

**Comment by AlvSovereign** [comment]
Hi FrDH

I got it to work a couple days ago.

How I got it to work was to remove all the options in the .tosrus({}) inline JS code.

Oddly enough, as soon as I put in any options, swiping stops working, and I suspect that I need to explicitly code for 'drag' in mobile.

Is this it's intended behaviour?

I have a separate issue, unrelated to this which I'll share later.

Thanks again for your time

AlvSovereign
